base/essentials.jl

```diff
@@ -712,6 +712,33 @@ function invokelatest(@nospecialize(f), @nospecialize args...; kwargs...)
     Core._apply_latest(inner)
 end
 
+"""
+   invokefrozen(f, rt, args...; kwargs...)
```

Comment:
```suggestion
    invokefrozen(f, rt, args...; kwargs...)
```
Markdown requires 4 spaces of indentation to recognize code blocks.